backend/kaos_backend/util/validators.py

```diff
@@ -82,19 +85,41 @@ def validate_bundle_structure(cls, directory, req_files):
                     for f in req_files:
                         cls.validate_file(f, files)
 
+                    if mode:
+                        for file in files:
+                            if file == mode:
+                                file_path = os.path.join(model_dir, mode)
+                                cls.validate_is_file_executable(file_path, mode)
+
+    @classmethod
+    def validate_is_file_executable(cls, executable_file, mode):
+        f = open(executable_file)
+        first_line = f.readline()
+        if cls.SHEBANG not in first_line:
```

Comment:
Could we use regular expressions in order to be sure that the #! is in the beginning of the line? 
Something like: 
```bash
if re.match("^#!", first_line) is None:
    raise ....
``` 
In this case `cls.SHEBANG` will not be used anywhere

Reply:
Good idea. Have now introduced regex to validate the first line